src/libxtp/gwbse/gwbse.cc

```diff
@@ -197,6 +197,15 @@ void GWBSE::Initialize(tools::Property& options) {
             options.ifExistsReturnElseReturnDefault<int>(key + ".exctotal", _bseopt.nmax);
     if (_bseopt.nmax > bse_size || _bseopt.nmax < 0) _bseopt.nmax = bse_size;
 
+    _bseopt.davidson =
+            options.ifExistsReturnElseReturnDefault<bool>(key + ".davidson", _bseopt.davidson);
+
+    _bseopt.jocc =
```

Comment:
make suboption and only use if _bseopt.davidson exists

key+".davidson.jocc"

but the names do not explain well what is supposed to happen